proxy.go

```diff
@@ -102,6 +102,13 @@ func (p *Proxy) init(front, back, capture Socket) {
 		}
 	)
 
+	// Subscribe all. If we don't do this, the sub socket will drop all messages.
+	//  It has no effect for other socket types.
+	if err := front.SetOption(OptionSubscribe, ""); err != nil {
+		log.Fatalf("Subscribe error: %v\n", err)
```

Comment:
perhaps return an error instead of fatal-ing?
this would involve changing the signature of `Proxy.init` and `NewProxy`... but ok.

Reply:
Updated in branch issue-108.